Makefile

```diff
@@ -23,7 +23,7 @@ ifdef SUITE
 ifndef GITHUB_ACTIONS
 	-@make lint
 endif
-	@bash test/scripts/test_suite.sh ${SUITE}
+	@bash test/scripts/test_suite.sh ${SUITE} ${CHANNEL}
```

Comment:
Please update readme to demonstrate how to locally test

Reply:
thanks, done